groundstation/static/js/components/Countdown.jsx

```diff
@@ -153,6 +153,16 @@ class Countdown extends Component{
                     nextPassover: data.nextPassover
                 });
             });
+
+            // temporary place to send slack messages for passover
+      
+            fetch('/api/passovers', {
+              method: 'HEAD',
+              headers: {
+                'Content-Type': 'application/json',
+                'Authorization': 'Bearer '+ localStorage.getItem('auth_token')
+              }
+            });
         }
 	}
```

Comment:
Should there be two closing parenthesis here?

```suggestion
```

Reply:
Looking at master it seems those parenthesis are there too, I think the indentation is off for the function so the last one needs to be pushed back a tab.